server/channels/api4/channel.go

```diff
@@ -1537,6 +1538,37 @@ func viewChannel(c *Context, w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func viewMultipleChannels(c *Context, w http.ResponseWriter, r *http.Request) {
+	c.RequireUserId()
+
+	auditRec := c.MakeAuditRecord("viewMultipleChannels", audit.Fail)
+	defer c.LogAuditRec(auditRec)
```

Comment:
We typically don't create audit records for read API events. There is a separate ticket that's meant to track adding audit records for all read events, but let's not add audit logging here prematurely.